packages/react/package.json

```diff
@@ -5,6 +5,7 @@
   "module": "dist/esm/index.js",
   "exports": {
     ".": "./dist/esm/index.js",
+    "./legacy": "./dist/esm/legacy.js",
```

Comment:
Despite `legacy/package.json`, ESM treats `exports` as an allowlist.

Reply:
Wow, I'eve never seen this, interesting